src/overlays/actors/ovl_Elf_Msg/z_elf_msg.c

```diff
@@ -180,7 +180,7 @@ void ElfMsg_CallNaviCuboid(ElfMsg* this, GlobalContext* globalCtx) {
         ((player->actor.posRot.pos.y - this->actor.posRot.pos.y) < (100.0f * this->actor.scale.y)) &&
         (fabsf(player->actor.posRot.pos.z - this->actor.posRot.pos.z) < (100.0f * this->actor.scale.z))) {
         player->naviMessageId = ElfMsg_GetMessageId(this);
-        navi->unk_298 = &this->actor;
+        navi->elfMsg = &this->actor;
```

Comment:
```suggestion
        navi->elfMsg = this;
```
since `ElfMsg* elfMsg;`
also in one other place in z_elf_msg.c